librosa/core/pitch.py

```diff
@@ -430,6 +430,16 @@ def yin(y, sr=22050, frame_length=2048, win_length=1024, hop_length=None,
     y : np.ndarray [shape=(n,)]
         audio time series.
 
+    fmin: number > 0 [scalar]
+        minimum frequency in Hertz.
+        The recomended minimum is librosa.note_to_hz('C2') (~65 Hz)
+        though lower values may be feasible.
+
+    fmax: number > 0 [scalar]
+        maximum frequency in Hertz.
+        The recomended maximum is librosa.note_to_hz('C7') (~2093 Hz)
+        though higher values may be feasible.
+
     sr : number > 0 [scalar]
         sampling rate of `y` in Hertz.
```

Comment:
Code formatting should use double-backticks: ` ``y`` `, not single backticks ` `y` `.

The exception is when formatting a function name that should be xrefed/hyperlinked in the documentation, like:

```
See `librosa.stft` for details
```